### PR TITLE
Weight Edge case and Constant Speed Internal Combustion Engine

### PR DESCRIPTION
**Pull request by planes**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: trunk/SUAVE/Components/Energy/Networks/__init__.py [hunk 1]
```diff
@@ -4,19 +4,20 @@
...
+from .Internal_Combustion_Propeller                import Internal_Combustion_Propeller
+from .Lift_Cruise                                  import Lift_Cruise
+from .Serial_Hybrid_Ducted_Fan                     import Serial_Hybrid_Ducted_Fan
+from .Vectored_Thrust                              import Vectored_Thrust
+from .Propulsor_Surrogate                          import Propulsor_Surrogate
+from .Battery_Propeller                            import Battery_Propeller
+from .Ramjet                                       import Ramjet
+from .Scramjet                                     import Scramjet
+from .Liquid_Rocket                                import Liquid_Rocket
+from .Internal_Combustion_Propeller_Constant_Speed import Internal_Combustion_Propeller_Constant_Speed
```
Comment (mclarke2) [inline trunk/SUAVE/Components/Energy/Networks/__init__.py:23]: What do you mean by constant speed? like constant rpm? Also is this another type of network or just a variant of the ICE network. If so, we can look to merge the two

**Review by mclarke2 — COMMENTED** [review]
Please see comment in __init__.py Network file

File: trunk/SUAVE/Components/Energy/Networks/__init__.py (hunk 1 above)
Comment (planes) [reply trunk/SUAVE/Components/Energy/Networks/__init__.py:23]: Constant speed is constant RPM. Constant Speed is the industry accepted term for constant RPM. The two methods rely on different engine calcs. So basically the networks are backwards of each other, CS goes prop->engine, Fixed pitch goes engine->prop. These networks are different enough I believe they should exist separately, and there is a clear need for both.

**Review by mclarke2 — APPROVED** [review]
Comments Reviewed. Approved

**Pull request merged**